description.md

```diff
@@ -20,6 +20,25 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+The primary goal of drills is to promote mastery and internalisation of skills or knowledge.
+By repeating tasks or exercises, students can reinforce their understanding and enhance their ability to apply learned concepts effectively.
+However, it's important to balance drills with other types of learning activities to maintain student engagement and prevent monotony.
+
```

Comment:
It would be helpful to explain the difference between drills and practical sessions/homework.

Reply:
I added some phrases about this.
Let me know if it's good.